src/Sulu/Bundle/SecurityBundle/Resources/js/containers/Permissions/Permissions.js

```diff
@@ -0,0 +1,224 @@
+// @flow
+import React, {Fragment} from 'react';
+import {action, autorun, computed, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import {Loader} from 'sulu-admin-bundle/components';
+import {MultiSelect} from 'sulu-admin-bundle/containers';
+import securityContextsStore from '../../stores/SecurityContextsStore';
+import type {SecurityContextGroups, SecurityContexts} from '../../stores/SecurityContextsStore/types';
+import type {ContextPermission} from './types';
+import permissionsStyle from './permissions.scss';
+import PermissionMatrix from './PermissionMatrix';
+
+type Props = {
+    system: string,
+    onChange: (value: Array<ContextPermission>) => void,
+    value: Array<ContextPermission>,
+};
+
+@observer
+export default class Permissions extends React.Component<Props> {
+    static webspacePlaceholder = '#webspace#';
+
+    systemDisposer: () => void;
+
+    @observable securityContextGroups: SecurityContextGroups;
+
+    @action componentDidMount() {
+        this.systemDisposer = autorun(
+            () => {
+                securityContextsStore.loadSecurityContextGroups(this.system).then(action((securityContextGroups) => {
```

Comment:
Until now we've named the stores in singular (e.g. `WebspaceStore`, `ResourceMetadataStore`, ...)